APE10.rst

```diff
@@ -0,0 +1,178 @@
+Roadmap for Python 3-only support
+---------------------------------
+
+author: Tom Robitaille
+
+date-created: 2016 July 13
+
+date-last-revised: 2016 July 13
+
+type: Process
+
+status: Draft
+
+Abstract
+--------
+
+The purpose of this APE is to set out a roadmap for future releases of the
+Astropy core package, in particular to identify the process for switching to
+releases that support only Python 3.
+
+Detailed description
+--------------------
+
+A number of scientific Python projects, including IPython, Matplotlib, Sympy,
+and others have now signed the `Python 3 statement
+<https://python3statement.github.io>`_ which states that scientific Python
+packages will drop support for Python 2 no longer than 2020. In fact, the
+IPython developers have now released `IPython 5.0
+<http://blog.jupyter.org/2016/07/08/ipython-5-0-released/>`_, which will be the
+last major release to support Python 2 - since it is a long term support (LTS)
+release, there will be 5.x bug fix releases until some time in 2019, but future
+major releases that add new features, starting with 6.0 (in 2017), will support
+only Python 3. This APE proposes that the Astropy project should also sign the Python 3 statement
+statement, and provides a target roadmap for this to happen.
+
+Roadmap
+-------
+
+The proposed roadmap for releases, including past releases for reference, is the
+following:
+
+==========   ==================  ==========================================
+Version      Release date        Notes
```

Comment:
`Release date` -> `(Planned) Release date`?  b/c some are already done and some are future.  Maybe also mark which ones have happened as of this APE just so that no one looking at this in the future gets concerned if it seems the dates here don't quite match the reality of future releases.